interface_config.js

```diff
@@ -39,8 +39,6 @@ var interfaceConfig = {
 
     DISABLE_DOMINANT_SPEAKER_INDICATOR: false,
 
-    DISABLE_FOCUS_INDICATOR: false,
```

Comment:
Instead of deleting it, comment it out and put a comment indicating it's deprecated and the option name in config.js.